Intro-JS/homework/homework.js

```diff
@@ -27,94 +29,155 @@ const newModulo = 21 % 5 === null ;
 
 function returnString(str) {
   //simply return the string provided: str
+  const zaarab = str;
```

Comment:
str is the variable name and zaarab the value to be assigned. the return statement should include str which has zaarab attached to it. please review. thanks